### DISCUSSION TITLE
fast and furious boot time

### DISCUSSION THREAD

**Discussion by fachos** [devmfc/debian-on-amlogic#33] [Q&A] [answered]
Hi dev,

Unbelievable boot take only 7second most box i try...
im surprised, good job MR DEV... i have small request,
can you activate the binfmt_misc in kernel next release? 
 
thank you

**Comment by devmfc** [comment]
CONFIG_BINFMT_MISC is considered insecure by some hardening tutorials. (although I don't see a big risk)
Why do you need it to be enabled?

**Comment by fachos** [comment]
Hi..

i try to run docker but some file not for arm.. it build for amd64...

**Reply by devmfc** [reply]
?? And you think CONFIG_BINFMT_MISC  will solve that? These boxes are arm64, you should not run amd64 binaries on them?

**Comment by fachos** [comment]
Hi 
Some docker files specifically for amd64, for sure cannot run under arm cpu but with binfmt enabled.. it shouldnt be a problem.. base on my experience

**Reply by devmfc** [reply, marked as answer]
Ok, attached kernel has the option enabled. Can you try it and if it works, let me know and I will enable it in the next release.

`dpkg -i ./linux-image-6.6.20-meson64_20240302_arm64.deb`

[linux-image-6.6.20-meson64_20240302_arm64.deb.zip](https://github.com/devmfc/debian-on-amlogic/files/14472911/linux-image-6.6.20-meson64_20240302_arm64.deb.zip)

**Comment by fachos** [comment]
Thank you dev…

will give a try tonight… let you know soon

**Comment by fachos** [comment]
Hi dev....

work like a charm...

before kernel update :- 
docker logs show
```
exec /bin/sh: exec format error
exec /bin/sh: exec format error
exec /bin/sh: exec format error
```

after i update the kernel:-
docker logs show
```
process.platform : linux
docker-031d7e368ed0fb92
PeerService(1896) -> creating peer response -> 200
start connection
P2PS-MainSocket -> Authentication
PeerService(1896) -> Connection established
markPeerAsAlive -> isAlive
peer connected
```
work like a charm
thank you so much....
